nav2_bt_navigator/behavior_trees/navigate_w_recovery_retry.xml

```diff
@@ -0,0 +1,21 @@
+<root main_tree_to_execute="BehaviorTree">
+    <!-- ////////// -->
+    <BehaviorTree ID="BehaviorTree">
+ 		<RetryUntilSuccesful name="retry_navigate" num_attempts="7"> 
+            <FallbackStar>
+                <SequenceStar name="navigate">
+          	        <ComputePathToPose endpoints="${endpoints}" path="${path}"/>
+	        		<FollowPath path="${path}"/>                    
+                </SequenceStar>
+                <ForceFailure>
+                  <SequenceStar name="recovery">
+                    <BackUp/>
```

Comment:
I feel hesitant to include BackUp in the recovery behavior without a collision checker. This behavior was also not part of the default recovery behaviors in MoveBase, so I'd recommend we remove it for now.